blocks/recent-posts/recent-posts.js

```diff
@@ -0,0 +1,36 @@
+import { getAllBlogs } from '../../scripts/scripts.js';
+import { createOptimizedPicture, getMetadata } from '../../scripts/lib-franklin.js';
+
+function createCard(row, style) {
+  const card = document.createElement('div');
+  if (style) card.classList.add(style);
+  if (row.image && row.title) card.prepend(createOptimizedPicture(row.image));
+  const cardContent = document.createElement('div');
+  cardContent.classList.add('card-content');
+  const link = document.createElement('a');
+  link.classList.add('blog-link');
+  link.href = row.path;
+  if (row.title) link.innerHTML = `<h5>${row.title}</h5>`;
```

Comment:
Change this 'h5' to an 'h3' to improve the Google accessibility score.
Google says: 
![image](https://github.com/mkclawson/impala/assets/3883795/05cba36a-f38d-4ccd-b1a5-e7faa7f4ebb8)

Reply:
fixed